index.html

```diff
@@ -6,6 +6,175 @@
+    <main>
+        <section>
+            <div>
+                <h1>The Design Thinking</h1>
+                <p>Tools, tutorials, design and innovation experts, all in one place! The most intuitive way to imagine your next user experience.</p>
+            </div>
+            <div>
+                <div>
+                    <a href="#">Ger started</a>
+                </div>
+                <div>
+                    <img src="icons/Triangle.png" alt="triangle" srcset="">
+                    <a href="#">Watch the Video</a>
+                </div>
+                <div>
+                    <img src="images/Image.png" alt="happy people" srcset="">
+                </div>
+            </div>
+        </section>
+        <section>
+            <div>
+                <h2>We design tools to unveil your superpowers</h2>
+            </div>
+            <div>
+                <ul>
+                    <li>
+                        <img src="icons/icon1.svg" alt="icon" srcset="">
+                        <h4>First click tests</h4>
+                        <p>While most people enjoy casino gambling,</p>
+                    </li>
+                    <li>
+                        <img src="icons/icon2.svg" alt="icon" srcset="">
+                        <h4>Design surveys</h4>
+                        <p>Sports betting, lottery and bingo playing for the fun</p>
+                    </li>
+                    <li>
+                        <img src="icons/icon3.svg" alt="icon" srcset="">
+                        <h4>Preference tests</h4>
+                        <p>The Myspace page defines the individual.</p>
+                    </li>
+                    <li>
+                        <img src="icons/icon4.svg" alt="icon" srcset="">
+                        <h4>Five second tests</h4>
+                        <p>Personal choices and the overall personality of the person. </p>
+                    </li>
+                </ul>
+            </div>
+            <div>
+                <a href="#">SIGN UP NOW</a>
+            </div>
+        </section>
+        <section>
+            <div>
+                <p>Effortless Validation for</p>
+                <h2>Design Professionals</h2>
+                <p>The Myspace page defines the individual, his or her characteristics, traits, personal choices and the overall personality of the person. </p>
+            </div>
+            <div>
+                <h4>Accessory makers</h4>
+                <p>While most people enjoy casino gambling, sports betting, lottery and bingo playing for the fun</p>
+            </div>
+            <div>
+                <h4>Alterationists</h4>
+                <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
+            </div>
+            <div>
+                <h4>Custom Design designers</h4>
+                <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
+            </div>
```

Comment:
тут треба замінити h4 => h3

```suggestion
            <div>
                <h3>Accessory makers</h3>
                <p>While most people enjoy casino gambling, sports betting, lottery and bingo playing for the fun</p>
            </div>
            <div>
                <h3>Alterationists</h3>
                <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
            </div>
            <div>
                <h3>Custom Design designers</h3>
                <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
            </div>
```